src/arrayMethodSort.js

```diff
@@ -4,8 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a.toString() > b.toString()) {
+        return 1;
+      } else if (a.toString() < b.toString()) {
+        return -1;
+      } else {
+        return 0;
+      }
+    },
+  ) {
+    let tempValue;
+
+    for (let i = 0; i < this.length; i++) {
+      for (let k = 0; k < this.length - 1; k++) {
```

Comment:
The inner loop can iterate until `this.length - i - 1` instead of `this.length - 1` as the largest element will be at the end after each iteration of the outer loop, and this can save unnecessary comparisons.